cypress/integration/portal-dashboard/portal-dashboard-feedback.spec.js

```diff
@@ -86,7 +86,7 @@ context("Portal Dashboard Feedback Panel", () => {
     });
   });
   context('Feedback Rows', () => {
-    describe('verify activity-level feedback textareas appear and accept input', () => {
+    describe('verify activity-level feedback appear and accept input', () => {
```

Comment:
This is a very picky comment on a tiny typo, but I think it'd be good to change "appear" to "appears." Or maybe say something like "feedback elements" appear? Not critical though.